services/traction/api/db/migrations/versions/v1_invitations_12a486c50ac3.py

```diff
@@ -0,0 +1,82 @@
+"""v1-invitations
+
+Revision ID: 12a486c50ac3
+Revises: 0fbab19dd1f4
+Create Date: 2022-05-04 13:00:33.959764
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "12a486c50ac3"
+down_revision = "0fbab19dd1f4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "invitation",
```

Comment:
do we want to be more specific? connection_invitation? 

Just wondering if we will have a different kind of invitation at some point.

Reply:
multi-use do not result in contact. we can move the other invitations here too if that makes sense. but this is quite a different use case. no resulting contact until someone uses this invitation, so we need to track this and the logic (future assumption is hanging a proof request workflow off this) differently.  we most certainly can move the create-invitation/receive-invitation to store data in this table. might make more sense.

Reply:
hmm, that is a very good point about different invitations, assuming you mean something like a traction invitation for tenancy?